Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/vcs.xml	(date 1546095532210)
+++ .idea/vcs.xml	(date 1546095532210)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: src/MathMoon.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/MathMoon.java	(date 1545566011723)
+++ src/MathMoon.java	(date 1545566011723)
@@ -0,0 +1,17 @@
+import java.util.Scanner;
+
+public class MathMoon {
+    public static void main  (String [] args)
+    {
+        System.out.println("Эта программа  поможет узнать ваш вес на Луне, в кг");
+        System.out.println();
+        System.out.println("Введите ваш вес");
+
+        Scanner keyboard = new Scanner(System.in);
+        double wpe  = keyboard.nextDouble();
+        double wpm = wpe*0.17;
+        System.out.println("Ваш вес на луне составляет " + wpm + "кг");
+
+    }
+
+}
